cinch/roles/jenkins_master/tasks/configure.yml

```diff
@@ -23,5 +23,18 @@
     group: jenkins
     state: directory
 
+- name: set hostname to match inventory hostname
+  hostname:
+    name: "{{ inventory_hostname }}"
+  when: jenkins_ssl_cert is not defined
```

Comment:
Sure. The point is to avoid the error message produced when the hostname of the machine doesn't match the certificate. In that case, the 'ensure_up' task will timeout because Python will fail to verify the SSL cert and therefore successfully connect to the host.